textile-utils-api-v1/build.gradle

```diff
@@ -0,0 +1,10 @@
+archivesBaseName = 'textile-utils-api-v1'
+version = '1.0.0'
+
+dependencies {
+	implementation project(':textile-api-base')
```

Comment:
Unnecessary dependency

Reply:
It is unnecessary.

Reply:
actually keep this but move `Identifier` to api base